src/examples/zkapps/voting/member.ts

```diff
@@ -19,36 +19,19 @@ let dummyWitness = Array.from(Array(MyMerkleWitness.height - 1).keys()).map(
 );
 
 export class Member extends CircuitValue {
```

Comment:
I had to trim a few unnecessary fields here to get within the transaction limits